tailfs/compositefs/openfile.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package compositefs
+
+import (
+	"context"
+	"io/fs"
+	"os"
+
+	"github.com/tailscale/xnet/webdav"
+	"tailscale.com/tailfs/shared"
+	"tailscale.com/util/pathutil"
+)
+
+func (cfs *compositeFileSystem) OpenFile(ctx context.Context, name string, flag int, perm os.FileMode) (webdav.File, error) {
+	if pathutil.IsRoot(name) {
```

Comment:
Consider maybe inverting the sense of this, so the bulk of the function can be one level less indented. Just a legibility thing, it's fine as it is functionally.